### PR TITLE
add support for discovery of cpu-partitioning via additional means (i…

### PR DESCRIPTION
**Pull request by k-rister**
…solcpus, nohz_full)

### CODE REVIEW

File: discover-cpu-partitioning.py [hunk 1]
```diff
@@ -86,6 +86,54 @@ def get_rcu_nocbs():
...
+    if path.exists() and path.is_file():
+        with path.open() as fh:
+            kernel_cli = fh.readline().rstrip()
+            t_global.log.debug("get_isolcpus: kernel_cli: %s" % (kernel_cli))
+
+            # find 'isolcpus=[<arg>[,<arg>[,]]]<cpu-list>' if it exists in the kernel command line
+            match = re.search(r"isolcpus=(\w+,)*([0-9\-,]+)", kernel_cli)
+            if match:
+                t_global.log.debug("get_isolcpus: %s" % (match.group(2)))
+                isolcpus = system_cpu_topology.parse_cpu_list(match.group(2))
```
Comment (rafaelfolco) [inline discover-cpu-partitioning.py:105]: else:
     t_global.log.debug("no isolcpus found..."

**Review by rafaelfolco — APPROVED** [review]
lgtm

File: discover-cpu-partitioning.py (hunk 1 above)
Comment (k-rister) [reply discover-cpu-partitioning.py:107]: I added a similar debug message to all 3 functions.

**Review by rafaelfolco — APPROVED** [review]
lgtm

**Pull request merged**